### DISCUSSION TITLE
検索画面の動作（[検索オプション]から[検索ワード入力欄] への転写）

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#360] [要望・アイデア・不具合]
検索画面に表示されている「検索オプション」一覧の各項目をクリックすると、それがそのまま「検索ワード入力欄」に転写されるのが仕様になっていると思われます（経験的に）。

　ところが、「created:」だけは、**転写された後で**、「検索オプション」に表示されている日付の**前日の日付に変わってしまいます**。数日前までは、そのような現象は見られませんでした。バグではないでしょうか？

これに関係しているかもしれませんので、以下、**追加情報**です。

　以前に#305で、「指定した期間分だけでなく、それ以前の記事まで検索出力される」という報告をしました。これは既に修正済と聞いていますが、その修正以前に、もう一つ気になる現象（下記）が見つかっていたのですが未報告でした。

> 前回の報告時には、検索画面での日付だけを見て「指定期間外」の誤検索と判断していたのですが、いざ、その記事を開いて記事内の投稿期日を見ると、正しく「指定期間内」に入っていることが分かりました。要は、**検索ページの作成日付と実際の記事の作成日付が合っていなかった**ということです。

**Comment by mkdk09** [comment]
@tsubolabo 
ご報告ありがとうございます！
一度調査いたします🙏

**Reply by tsubolabo** [reply]
@mkdk09 

　今日の午前中は初回の投稿のような状況でしたが、今（15:00）見ると正常です。もう修正されたのですか？

**Reply by tsubolabo** [reply]
また再現しました。どうも、早朝に起こる現象のようです。

**Reply by yoshinoyuya** [reply]
@tsubolabo さん
恐らくなのですが、タイムゾーンの違いから発生したバグだと思います。
そのため、特定の時間だと正常に動いているようにみえるのだと思います。

**Reply by mkdk09** [reply]
@tsubolabo
追加の情報ありがとうございます！
現在調査中ですので改めてお返事いたします🙏

@yoshinoyuya
代わりに返答していただきありがとうございます🙏
ご指摘の通りタイムゾーンの違いから発生していると考えております🙇‍

**Comment by mkdk09** [comment]
@tsubolabo 
> 「created:」だけは、転写された後で、「検索オプション」に表示されている日付の前日の日付に変わってしまいます。

こちらの不具合の修正が完了しました！
ご不便をおかけしました🙇‍
